src/code/z_game_dlftbls.c

```diff
@@ -9,10 +9,10 @@
     { NULL, 0, 0, NULL, NULL, NULL, init, destroy, NULL, NULL, 0, size }
 
 GameStateOverlay gGameStateOverlayTable[] = {
-    GAMESTATE_OVERLAY_INTERNAL(TitleSetup_Init, TitleSetup_Destroy, sizeof(GameState)),
-    GAMESTATE_OVERLAY(select, Select_Init, Select_Destroy, sizeof(SelectContext)),
-    GAMESTATE_OVERLAY(title, Title_Init, Title_Destroy, sizeof(TitleContext)),
+    GAMESTATE_OVERLAY_INTERNAL(Setup_Init, Setup_Destroy, sizeof(GameState)),
```

Comment:
```suggestion
    GAMESTATE_OVERLAY_INTERNAL(Setup_Init, Setup_Destroy, sizeof(SetupState)),
```
Since this is now using `SetupState` instead of `GameState`